oraichain/assetlist.json

```diff
@@ -194,18 +194,40 @@
       "type_asset": "cw20",
       "address": "orai12hzjxfh77wl572gdzct2fxv2arxcwh6gykc7qh",
       "base": "cw20:orai12hzjxfh77wl572gdzct2fxv2arxcwh6gykc7qh",
+      "traces": [
+        {
+          "type": "bridge",
+          "counterparty": {
+            "chain_name": "ethereum",
+            "base_denom": "0xdac17f958d2ee523a2206206994597c13d831ec7"
+          },
+          "provider": "Oraibridge"
+        },
+        {
+          "type": "ibc",
+          "counterparty": {
+            "chain_name": "oraibridge",
```

Comment:
I wasn't aware it was bridged to another chain and then IBC transferred here. If Oraibridge is another chain it will need to be registered. But I'm confused why an IBC transferred token doesn't result in a ibc/hash denom on this chain, but instead in a cw20 address...

Reply:
OraiBridge is the chain to transfer funds from Oraichain to EVM ( Oraichain -> OraiBridge -> Bitcoin )
Can i register that to the next pull request?